enterprise/internal/a8n/runner.go

```diff
@@ -113,17 +113,15 @@ func NewRunnerWithClock(store *Store, ct CampaignType, search repoSearch, defaul
 // The time after which a CampaignJob's execution times out
 const jobTimeout = 2 * time.Minute
 
-// Run executes the CampaignPlan by searching for relevant repositories using
-// the CampaignType specific searchQuery and then executing CampaignJobs for
-// each repository.
-// Before it starts executing CampaignJobs it persists the CampaignPlan and the
-// new CampaignJobs in a transaction.
+// Persist persists the CampaignPlan and associated CampaignJobs by searching for relevant repositories using
+// the CampaignType specific searchQuery.
 // What each CampaignJob then does in each repository depends on the
 // CampaignType set on CampaignPlan.
 // This is a non-blocking method that will possibly return before all
```

Comment:
I don't think this line makes sense anymore:

```suggestion
// 
```

I would re-order remaining sentences in the comment to follow this structure:

```
// Persist persists the CampaignPlan ...
// CampaignJobs will be picked up by a background process ...
// What each CAmpaignJob then does in each repository ...
```